shared/src/stripe/StripeEventHandler.ts

```diff
@@ -180,8 +180,8 @@ export class StripeEventHandler {
 	 * This is mainly for failed payments where we didn't create a user through the website directly
```

Comment:
@mkue, is this comment actually true? 
In the webhooks logs from Stripe, it seems that the user doesn't exist for a successful twint payment. Where do you create the user? Is it a race condition, whatever is faster?

Reply:
No, this comment is wrong. The usual way for a user to get created is through the Stripe webhook.

And yes, it's technically a race condition. If a new user fills out the CreateUserForm before the Stripe webhook has been called (which should only happen if Stripe has a problem), then the user is created when the form is submitted. This is a very unlikely case, though.

I don't see how this can be avoided as we need to store the user info on form submit.